spec/build_spec.rb

```diff
@@ -263,9 +268,9 @@ def pull_request?
       allow(Octokit::Client).to receive(:new).and_return(DummyClient2.new)
       allow(PotentialBuild).to receive(:new).and_return(true)
       b = Build.new('abcdef', 'spec/resources', 10)
-      expect(b.client.branches('', 1).length).to eql 6 # should have four total total branches
+      expect(b.client.branches('', 1).length).to eql 8 # this is the absolute total
       b.query_branches
-      expect(b.potential_builds.length).to eql 2 # but only two are valid and new enough to build
+      expect(b.potential_builds.length).to eql 3 # this is the number of expected valid ones
```

Comment:
One of the new ones I added should actually be included in the list -- the develop branch.